pkg/apis/kubeflow.org/v1/pytorch_types.go

```diff
@@ -20,23 +20,23 @@ import (
 )
 
 const (
-	// PytorchJobDefaultPortName is name of the port used to communicate between Master and
+	// PyTorchJobDefaultPortName is name of the port used to communicate between Master and
 	// workers.
-	PytorchJobDefaultPortName = "pytorchjob-port"
-	// PytorchJobDefaultContainerName is the name of the PyTorchJob container.
-	PytorchJobDefaultContainerName = "pytorch"
-	// PytorchJobDefaultPort is default value of the port.
-	PytorchJobDefaultPort = 23456
-	// PytorchJobDefaultRestartPolicy is default RestartPolicy for PyTorchReplicaSpec.
-	PytorchJobDefaultRestartPolicy = RestartPolicyOnFailure
-	// PytorchJobKind is the kind name.
-	PytorchJobKind = "PyTorchJob"
-	// PytorchJobPlural is the PytorchPlural for pytorchJob.
-	PytorchJobPlural = "pytorchjobs"
-	// PytorchJobSingular is the singular for pytorchJob.
-	PytorchJobSingular = "pytorchjob"
-	// PytorchJobFrameworkName is the name of the ML Framework
-	PytorchJobFrameworkName = "pytorch"
+	PyTorchJobDefaultPortName = "pytorchjob-port"
+	// PyTorchJobDefaultContainerName is the name of the PyTorchJob container.
+	PyTorchJobDefaultContainerName = "pytorch"
+	// PyTorchJobDefaultPort is default value of the port.
+	PyTorchJobDefaultPort = 23456
+	// PyTorchJobDefaultRestartPolicy is default RestartPolicy for PyTorchReplicaSpec.
+	PyTorchJobDefaultRestartPolicy = RestartPolicyOnFailure
+	// PyTorchJobKind is the kind name.
+	PyTorchJobKind = "PyTorchJob"
+	// PyTorchJobPlural is the PyTorchPlural for pytorchJob.
+	PyTorchJobPlural = "pytorchjobs"
+	// PyTorchJobSingular is the singular for pytorchJob.
+	PyTorchJobSingular = "pytorchjob"
+	// PyTorchJobFrameworkName is the name of the ML Framework
+	PyTorchJobFrameworkName = "pytorch"
```

Comment:
Does this have to be "PyTorch" too?

Reply:
@terrytangyuan I think that we need not change both values since the singular is defined as a lowercase in general, and all framework names are defined as a lower case such as `tensorflow`:

CRD doc: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/

https://github.com/kubeflow/training-operator/blob/de7c73ef6fdf7b8e3283bfd027b9dc07f6400400/pkg/apis/kubeflow.org/v1/tensorflow_types.go#L38